easybuild/easyblocks/s/sas.py

```diff
@@ -34,6 +34,19 @@
 class EB_SAS(EasyBlock):
     """Support for building/installing SAS."""
 
+    def __init__(self, *args, **kwargs):
+        """Custom constructor for SAS easyblock, initialize custom class variables."""
+
+        super(EB_SAS, self).__init__(*args, **kwargs)
+
+        # Use default SAS Installation Data File path
+        self.license_file = ''
+
+        # Set custom SAS Installation Data File path if defined and existing
+        if self.cfg['license_file'] is not None and os.path.isfile(self.cfg['license_file']):
```

Comment:
You can simplify this (the `is not None` is a bit silly, and otherwise you'll run into problems if `license_file` is set to `''`):

```python
if self.cfg['license_file'] and os.path.isfile(self.cfg['license_file']):

Reply:
That is something we should correct on other easyblocks since it is a recurring pattern.

Reply:
open an issue on it if you don't have time to dive in?